Stack/Opc.Ua.Core/Stack/Generated/Opc.Ua.DataTypes.cs

```diff
@@ -2803,6 +2803,7 @@ public new object MemberwiseClone()
     /// <exclude />
     [System.CodeDom.Compiler.GeneratedCodeAttribute("Opc.Ua.ModelCompiler", "1.0.0.0")]
     [DataContract(Namespace = Opc.Ua.Namespaces.OpcUaXsd)]
+    [Flags]
```

Comment:
this is a generated file, please remove [flags]

Reply:
Right. This code is imported from an external repository and generated by ModelCompiler